### PR TITLE
feat: improve drop share

### PR DESCRIPTION
**Pull request by alantoa**
# Why 

https://linear.app/showtime/issue/SHOW2-1519/improving-drop-shares 

# How 

I followed the Figma design to improve the QR code drop and fixed its size to avoid any unusual sizes on the web.

# Test
|iOS|Android|Web|Mobile Web|
|---|---|---|---|
|CleanShot 2023-03-13 at 3 57 38|CleanShot 2023-03-13 at 3 52 17|CleanShot 2023-03-13 at 4 06 32|CleanShot 2023-03-13 at 4 06 59|

# Finally Screenshot
|Before|After|
|---|---|
| |  |

### CODE REVIEW

**Review by hirbod — APPROVED** [review]
Looks awesome

File: apps/expo/.env.development [hunk 1]
```diff
@@ -1,30 +1,31 @@
-STAGE=development
-NEXT_PUBLIC_CHAIN_ID=mumbai
+STAGE=production
```
Comment (intergalacticspacehighway) [inline apps/expo/.env.development]: 😄

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM ❇️ after env changes 😄

File: apps/expo/.env.development (hunk 1 above)
Comment (alantoa) [reply apps/expo/.env.development]: sorry, it's for test, i have changed back in my local, will push it soon.

**Pull request merged**